### PR TITLE
fix(data-m): inconsistent alerts

### PR DESCRIPTION
**Pull request by alexkim205**
## Problem

Screen Shot 2022-08-24 at 11 56 46 AM

## Changes

Screen Shot 2022-08-24 at 2 20 04 PM

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/events/UsageDisabledWarning.tsx [hunk 1]
```diff
@@ -4,8 +4,7 @@ import React from 'react'
 export function UsageDisabledWarning({ tab }: { tab: string }): JSX.Element {
     return (
         <Alert
-            type="info"
-            showIcon
+            style={{ marginBottom: 16, display: 'flex', alignItems: 'center' }}
```
Comment (benjackwhite) [inline frontend/src/scenes/events/UsageDisabledWarning.tsx:7]: styles!! 👿 

Could we use classNames here instead? If not, could we quickly refactor this over to the [AlertMessage](https://storybook.posthog.net/?path=/docs/lemon-ui-alert-message--info) component which does support it?

Comment (alexkim205) [reply frontend/src/scenes/events/UsageDisabledWarning.tsx:7]: ah good callout!

**Review by pauldambra — APPROVED** [review]
💎

**Comment by pauldambra** [comment]
cherry-picked into 1.39.0

**Pull request merged**